Build on F32 in Copr is failing because of these two warnings. Not sure what's wrong.
```
In file included from /usr/include/glib-2.0/glib.h:113,
                 from /usr/include/glib-2.0/gobject/gbinding.h:28,
                 from /usr/include/glib-2.0/glib-object.h:22,
                 from /usr/include/glib-2.0/gio/gioenums.h:28,
                 from /usr/include/glib-2.0/gio/giotypes.h:28,
                 from /usr/include/glib-2.0/gio/gio.h:26,
                 from /usr/include/abrt/libabrt.h:14,
                 from /builddir/build/BUILD/abrt-java-connector-1.1.5/src/internal_libabrt.h:26,
                 from /builddir/build/BUILD/abrt-java-connector-1.1.5/src/abrt-checker.c:52:
/builddir/build/BUILD/abrt-java-connector-1.1.5/src/abrt-checker.c: In function 'callback_on_exception':
/usr/include/glib-2.0/glib/glib-autocleanups.h:28:3: error: 'stack_trace_str' may be used uninitialized in this function [-Werror=maybe-uninitialized]
   28 |   g_free (*pp);
      |   ^~~~~~~~~~~~
/builddir/build/BUILD/abrt-java-connector-1.1.5/src/abrt-checker.c:2480:30: note: 'stack_trace_str' was declared here
 2480 |             g_autofree char *stack_trace_str = generate_thread_stack_trace(jvmti_env, jni_env, tname, exception_object,
      |                              ^~~~~~~~~~~~~~~
In file included from /usr/include/glib-2.0/glib.h:113,
                 from /usr/include/glib-2.0/gobject/gbinding.h:28,
                 from /usr/include/glib-2.0/glib-object.h:22,
                 from /usr/include/glib-2.0/gio/gioenums.h:28,
                 from /usr/include/glib-2.0/gio/giotypes.h:28,
                 from /usr/include/glib-2.0/gio/gio.h:26,
                 from /usr/include/abrt/libabrt.h:14,
                 from /builddir/build/BUILD/abrt-java-connector-1.1.5/src/internal_libabrt.h:26,
                 from /builddir/build/BUILD/abrt-java-connector-1.1.5/src/abrt-checker.c:52:
/usr/include/glib-2.0/glib/glib-autocleanups.h:28:3: error: 'message' may be used uninitialized in this function [-Werror=maybe-uninitialized]
   28 |   g_free (*pp);
      |   ^~~~~~~~~~~~
/builddir/build/BUILD/abrt-java-connector-1.1.5/src/abrt-checker.c:2476:30: note: 'message' was declared here
 2476 |             g_autofree char *message = format_exception_reason_message(/*caught?*/NULL != catch_method,
      |                              ^~~~~~~
cc1: all warnings being treated as errors
```

--- a/src/configuration.c
+++ b/src/configuration.c
@@ -401,7 +401,7 @@ void parse_commandline_options(T_configuration *conf, char *options)
 void parse_configuration_file(T_configuration *conf, const char *filename)
 {
     /* Remains empty if any of the loading functions below fails */
-    map_string_t *settings = libreport_new_map_string();
+    map_string_t *settings = g_hash_table_new_full(g_str_hash, g_str_equal, g_free, g_free);
     if (filename[0] == '/')
     {
         libreport_load_conf_file(filename, settings, /*skip empty*/0);

--- a/src/configuration.c
+++ b/src/configuration.c
@@ -418,11 +418,11 @@ void parse_configuration_file(T_configuration *conf, const char *filename)
 
     map_string_iter_t iter;
     g_hash_table_iter_init(&iter, settings);
-    const char *key;
-    const char *value;
-    while(libreport_next_map_string_iter(&iter, &key, &value))
+    gpointer key;
+    gpointer value;
+    while(g_hash_table_iter_next(&iter, &key, &value))
     {
-        parse_key_value(conf, key, value, &ctx);
+        parse_key_value(conf, (char *)key, (char *)value, &ctx);
     }
 
     if (settings)

--- a/utils/abrt-action-analyze-java.c
+++ b/utils/abrt-action-analyze-java.c
@@ -139,7 +139,7 @@ work_out_list_of_remote_urls(struct sr_java_stacktrace *stacktrace)
         struct sr_java_frame *frame = thread->frames;
         while (NULL != frame)
         {
-            if (NULL != frame->class_path && libreport_prefixcmp(frame->class_path, "file://") != 0)
+            if (NULL != frame->class_path && g_str_has_prefix(frame->class_path, "file://") == 0)
             {
                 struct stat buf;
                 if (stat(frame->class_path, &buf) && errno == ENOENT)

